Xamarin.Forms.Core/VisualElement.cs

```diff
@@ -779,7 +805,7 @@ protected override void OnParentSet()
 			}
 #pragma warning restore 0618
 
-			FlowController.NotifyFlowDirectionChanged();
+			PropertyPropagationController.PropagatePropertyChanged(null);
```

Comment:
```suggestion
			PropertyPropagationController.PropagatePropertyChanged(VisualElement.FlowDirectionProperty.PropertyName);
```

Reply:
The purpose of the null is saying hey some event has happened at the parent level so all children you need to re-evaluate yourselves.

Visual and FlowDirection both propagate to the children so the children use the NULL to know that they need to check both properties of the parent

https://github.com/xamarin/Xamarin.Forms/pull/4449/files#diff-b2408192f3f7437c9eed488a33f903e8R1059

https://github.com/xamarin/Xamarin.Forms/pull/4449/files#diff-b2408192f3f7437c9eed488a33f903e8R1062